scan-command/src/main/java/io/ballerina/scan/utils/ScanUtils.java

```diff
@@ -452,39 +462,43 @@ private static Consumer<Toml> loadAnalyzers(ScanTomlFile scanTomlFile) {
     }
 
     /**
-     * Loads platforms to the {@link ScanTomlFile} representation of the Scan.toml file.
+     * Returns the {@link Boolean} value indicating whether the scan process should be continued.
      *
      * @param scanTomlFile the in-memory representation of the Scan.toml file
      * @param targetDir    the target directory
      * @param outputStream the print stream
+     * @return the boolean value indicating whether the scan process should be continued
      */
-    private static Consumer<Toml> loadPlatforms(ScanTomlFile scanTomlFile, Path targetDir, PrintStream outputStream) {
-        return platformTable -> {
+    private static Boolean loadPlatform(Toml platformTable, ScanTomlFile scanTomlFile, Path targetDir,
```

Comment:
let's return the primitive type boolean if possible here.

Reply:
Also check the place where this function is called too